Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.10-slim
```

Comment:
I'm not quite sure but I wonder if we have a slim version of 3.12? Maybe `3.12-slim`? The reason being is Vercel's compatibility with Python (I know Vercel doesn't support docker but I just wanna keep things consistent).

Reply:
I purposefully used 3.10 for being consistent with the Readme instruction which specifically stated to use Python 3.10.

![image](https://github.com/user-attachments/assets/46764ea7-6810-49a9-9a36-07a3d2454de0)